sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/QueryBuilderSamples.cs

```diff
@@ -57,6 +57,20 @@ public static void Main()
                 .Select("Room", "Temperature")
                 .From(AdtCollection.DigitalTwins)
                 .Build();
+
+            // SELECT * FROM DIGITALTWINS WHERE TEMPERATURE < 5
+            AdtQueryBuilder query6 = new AdtQueryBuilder()
+                .Select("*")
+                .From(AdtCollection.DigitalTwins)
+                .WhereComparison("Temperature", QueryComparisonOperator.LessThan, "6")
+                .Build();
+
+            // SELECT * FROM DIGITALTWINS WHERE IS_OF_MODEL('dtmi:example:room;1', exact)
+            AdtQueryBuilder query7 = new AdtQueryBuilder()
+                .Select("*")
+                .From(AdtCollection.DigitalTwins)
+                .WhereIsOfModel("dtmi:example:room;1", true)
```

Comment:
Did you experiment with the idea of having a Single `Where()` method call and then following it up with all the possible conditional methods?

` Where().IsOfModel("dtmi:example:room;1").IsTrue("Temperature", ">", 20) ... `

If so, what made you decide against that?

Reply:
Haven't experimented with that yet, it's on my current card though.

Reply:
It might be worthwhile to have another discussion session right after this PR to see which path looks the best moving forward. We can also discuss the logical operators and how we would want to support nested conditions in the same meeting ... to just discuss the conditional statements. Then we can do a redesign if applicable.